src/main/java/org/aleksey/springnew/controller/TodoController.java

```diff
@@ -0,0 +1,39 @@
+package org.aleksey.springnew.controller;
+
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.aleksey.springnew.dto.TodoCreateDto;
+import org.aleksey.springnew.dto.TodoHistoryResponseDto;
+import org.aleksey.springnew.dto.TodoResponseDto;
+import org.aleksey.springnew.dto.TodoUpdateDto;
+import org.aleksey.springnew.service.TodoService;
+import org.springframework.web.bind.annotation.*;
+import java.util.List;
+
+@RestController
+@RequestMapping("/todo")
```

Comment:
```suggestion
@RequestMapping("/todos")
```
please see: https://restfulapi.net/resource-naming/